internal/k8s/exec.go

```diff
@@ -37,6 +38,8 @@ type ExecParameters struct {
 }
 
 func (c *Client) execInPod(ctx context.Context, p ExecParameters) (*ExecResult, error) {
+	stdin := utils.NewCtrlCReader(ctx)
```

Comment:
This could be moved closer to where it's used first above the call to `req.VersionedParams()` below.

Reply:
It's actually needed even later, moved.